flixel/system/FlxSound.hx

```diff
@@ -442,11 +447,41 @@ class FlxSound extends FlxBasic
 	 */
 	public inline function stop():FlxSound
 	{
-		cleanup(autoDestroy, true, true);
+		cleanup(autoDestroy, true);
 		return this;
 	}
 	
 	/**
+	 * Sets the sound point at which to restart playing after a loop
+	 * @param	newloopPoint The new position in the sound, in milliseconds
+	 * @return	The updated value in milliseconds
+	 */
+	public function set_loopPoint(newloopPoint:Float):Float
+	{
+		if (looped && playing && loopPoint == 0.0)
+		{
+			loopPoint = newloopPoint;
+			var pos:Float = _channel.position;
+			cleanup(false, false);
+			startSound(pos);
+			return loopPoint;
+		}
+		loopPoint = newloopPoint;
```

Comment:
Hm, so it's a matter of having to manually manage the looping now... I wonder why we don't always do that then, seems less complicated. Either way, you should:

- create a helper method that handles this to avoid the redundancy with set_looped()
- add a comment to the call here explaning why it's necessary since it doesn't seem super-intuitive